assets/src/components/propertiesPanel/minischedule.tsx

```diff
@@ -546,16 +546,16 @@ const iconForDirectionOnLadder: (
   routeId: RouteId
 ) => ReactElement = (directionId, ladderDirections, routeId) => {
   if (directionId === null) {
-    return questionMarkIcon()
+    return questionMarkIcon("m-minischedule__svg--revenue")
```

Comment:
@skyqrose since we're using this string literal three times, perhaps it's time to extract it to a named constant.